Entities/Special/Dummy/Dummy.cfg

```diff
@@ -7,12 +7,13 @@
 $sprite_factory                            = generic_sprite
 
 @$sprite_scripts                           = Dummy.as;
-                      						 Wooden.as;
+                      				Wooden.as;
+						FireAnim.as;
 $sprite_texture                            = Dummy.png
 s32_sprite_frame_width                     = 24
 s32_sprite_frame_height                    = 24
 f32 sprite_offset_x                        = 0
-f32 sprite_offset_y                        = -4
+f32 sprite_offset_y                        = -8
```

Comment:
This makes their hitbox severely mismatch their sprite, which looks weird given that they can now be shot:
![image](https://user-images.githubusercontent.com/8986152/215292308-1d6f972a-1b25-457a-85b2-a8d8002afe65.png)

Not sure what would be the best way to fix this. The circle shape also makes stuck arrows look derpy.

Reply:
the proper way to fix this would be to set the sprite offset back to normal and then offset the blob's spawn position in map loading so the blob